src/App.tsx

```diff
@@ -1,16 +1,24 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+
+  const handleOnAdd = (movie: Movie) => {
```

Comment:
```suggestion
  const handleMovieAdd = (movie: Movie) => {
```
not a mistake but let's consider handlers naming according to naming convention. Here is an [article](https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364)